pkgs/tools/misc/chafa/default.nix

```diff
@@ -27,22 +30,30 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-9RkN0yZnHf5cx6tsp3P6jsi0/xtplWxMm3hYCPjWj0M=";
   };
 
+  outputs = [
+    "bin"
+    "dev"
+    "man"
+    "out"
+  ];
+
   nativeBuildInputs = [
     autoconf
     automake
     libtool
     pkg-config
     which
+    libavif
+    libjxl
+    librsvg
     libxslt
     libxml2
```

Comment:
Are you sure the libs should be in natibeBuildInputs? Normally they go to buildInputs

Reply:
Moved to `buildInputs` as suggested.